storage/mongo/utils.js

```diff
@@ -0,0 +1,16 @@
+var MongoClient = require( 'mongodb' ).MongoClient;
+var mydb;
+
+module.exports = function(config){
+
+  init = function ( callback ) {
+    MongoClient.connect( "mongodb://localhost:27017/AnthroLinkDB", function( err, db ) {
+      mydb = db;
```

Comment:
mongo utils is not supposed to have a init function. As this is a utility file there could be a lot of function but this is exported as a single module. Moreover instead of init function it should be connect function which would be called inside mongo/index.js